doc-src/SASS_REFERENCE.md

```diff
@@ -2534,6 +2534,53 @@ passed block are related to the other styles around where the block is defined.
       }
     }
 
+### Including a Mixin Dynamically
+
+There are times when it is convenient to call a mixin based on the
+result of a SassScript expression.
+
+For example, a library might want to delegate some styling to the
+application by allowing the application to register a mixin that the
+library would include where appropriate. Sometimes a content block is
```

Comment:
@nex3 We have this problem with `call` already. This is a common use case for that function. I think 4.0 will have to have a way of passing a function or mixin reference that isn't a string so the lookup can be done in one module context and used in another.

E.g. `function($function-name)` and `mixin($mixin-name)` would return an object of type `reference` which would include whatever internal binding information we need. The `call` function and `mixin` mixin would then be modified to accept references as well as strings. I would also note that this could enable #472 down the road.